pkg/ravencontroller/apis/calico/v3/blockaffinity.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
```

Comment:
Please add OpenYurt Copyright to the files in `pkg/ravencontroller/apis/calico/v3/` directory.
For example:
```
// Copyright 2020 The OpenYurt Authors.
// Copyright (c) 2019 Tigera, Inc. All rights reserved.
```

Reply:
The latest code has been commited. `Copyright 2020 The OpenYurt Authors` have been added.